### ISSUE TITLE
Build error on 0.5.0_R5

### ISSUE THREAD

**Issue by EwingKang** [intel/ros2_openvino_toolkit#78]
When building with `colcon build --symlink-install`
There will be errors preventing a success build:
```bash
--- stderr: dynamic_vino_lib                                 
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/standard_camera.cpp: In member function ‘virtual bool Input::StandardCamera::initialize()’:
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/standard_camera.cpp:25:28: error: ‘CV_CAP_PROP_FRAME_WIDTH’ was not declared in this scope
   setWidth((size_t)cap.get(CV_CAP_PROP_FRAME_WIDTH));
                            ^~~~~~~~~~~~~~~~~~~~~~~
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/standard_camera.cpp:26:29: error: ‘CV_CAP_PROP_FRAME_HEIGHT’ was not declared in this scope
   setHeight((size_t)cap.get(CV_CAP_PROP_FRAME_HEIGHT));
                             ^~~~~~~~~~~~~~~~~~~~~~~~
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/standard_camera.cpp: In member function ‘virtual bool Input::StandardCamera::initialize(int)’:
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/standard_camera.cpp:33:28: error: ‘CV_CAP_PROP_FRAME_WIDTH’ was not declared in this scope
   setWidth((size_t)cap.get(CV_CAP_PROP_FRAME_WIDTH));
                            ^~~~~~~~~~~~~~~~~~~~~~~
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/standard_camera.cpp:34:29: error: ‘CV_CAP_PROP_FRAME_HEIGHT’ was not declared in this scope
   setHeight((size_t)cap.get(CV_CAP_PROP_FRAME_HEIGHT));
                             ^~~~~~~~~~~~~~~~~~~~~~~~
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/standard_camera.cpp: In member function ‘virtual bool Input::StandardCamera::initialize(size_t, size_t)’:
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/standard_camera.cpp:44:13: error: ‘CV_CAP_PROP_FRAME_WIDTH’ was not declared in this scope
     cap.set(CV_CAP_PROP_FRAME_WIDTH, width);
             ^~~~~~~~~~~~~~~~~~~~~~~
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/standard_camera.cpp:45:13: error: ‘CV_CAP_PROP_FRAME_HEIGHT’ was not declared in this scope
     cap.set(CV_CAP_PROP_FRAME_HEIGHT, height);
             ^~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [CMakeFiles/dynamic_vino_lib.dir/src/inputs/standard_camera.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/video_input.cpp: In member function ‘virtual bool Input::Video::initialize()’:
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/video_input.cpp:33:28: error: ‘CV_CAP_PROP_FRAME_WIDTH’ was not declared in this scope
   setWidth((size_t)cap.get(CV_CAP_PROP_FRAME_WIDTH));
                            ^~~~~~~~~~~~~~~~~~~~~~~
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/video_input.cpp:34:29: error: ‘CV_CAP_PROP_FRAME_HEIGHT’ was not declared in this scope
   setHeight((size_t)cap.get(CV_CAP_PROP_FRAME_HEIGHT));
                             ^~~~~~~~~~~~~~~~~~~~~~~~
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/video_input.cpp: In member function ‘virtual bool Input::Video::initialize(size_t, size_t)’:
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/video_input.cpp:44:13: error: ‘CV_CAP_PROP_FRAME_WIDTH’ was not declared in this scope
     cap.set(CV_CAP_PROP_FRAME_WIDTH, width);
             ^~~~~~~~~~~~~~~~~~~~~~~
/home/ros/ros2_ws/src/ros2_openvino_toolkit/dynamic_vino_lib/src/inputs/video_input.cpp:45:13: error: ‘CV_CAP_PROP_FRAME_HEIGHT’ was not declared in this scope
     cap.set(CV_CAP_PROP_FRAME_HEIGHT, height);
             ^~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [CMakeFiles/dynamic_vino_lib.dir/src/inputs/video_input.cpp.o] Error 1
make[1]: *** [CMakeFiles/dynamic_vino_lib.dir/all] Error 2
make: *** [all] Error 2
---
Failed   <<< dynamic_vino_lib	[ Exited with code 2 ]

Summary: 10 packages finished [29.3s]
  1 package failed: dynamic_vino_lib
  1 package had stderr output: dynamic_vino_lib
  1 package not processed
```
and
```bash
  /home/ros/ros2_ws/src/ros2_openvino_toolkit/sample/src/image_object_client.cpp: In function ‘int main(int, char**)’:
/home/ros/ros2_ws/src/ros2_openvino_toolkit/sample/src/image_object_client.cpp:83:28: error: ‘cvPoint’ was not declared in this scope
       cv::rectangle(image, cvPoint(xmin, ymin), cvPoint(xmax, ymin + 20), cv::Scalar(0, 255, 0),
                            ^~~~~~~
```

**Comment by EwingKang** [comment]
I've submitted a pull request that fixed the issue.

**Comment by EwingKang** [comment]
I think this is fixed with [PR-79](#79) for the master 0.5.0_R5 branch. I'll close it.
